CMakeLists.txt

```
@@ -40,17 +43,29 @@ endforeach( OUTPUTCONFIG CMAKE_CONFIGURATION_TYPES )
 
 set(SC_MEMORY_SRC "${SC_MACHINE_ROOT}/sc-memory")
 
+if (${SC_USE_SANITAZER} EQUAL "address")
```

```suggestion
if (${SC_USE_SANITAZER} STREQUAL "address")
```
Fix it in places like this